go-apps/meep-platform-ctrl/server/user_authentication.go

```diff
@@ -25,19 +25,206 @@
 package server
 
```

Comment:
User activity should be logged in InfluxDB on session start and session end

Reply:
Will be addressed in another PR.